src/server/test/hotspot_partition_test.cpp

```diff
@@ -92,6 +114,30 @@ TEST_F(hotspot_partition_test, hotspot_partition_policy)
     test_rows[HOT_SCENARIO_1_WRITE_HOT_PARTITION].put_qps = 5000.0;
     expect_vector = {{0, 0, 0, 4, 0, 0, 0, 0}, {0, 0, 4, 0, 0, 0, 0, 0}};
     test_policy_in_scenarios(test_rows, expect_vector, calculator);
+    clear_calculator_histories(calculator);
+}
+
+TEST_F(hotspot_partition_test, send_hotkey_detect_request)
+{
+    const int READ_HOT_PARTITION = 7;
+    const int WRITE_HOT_PARTITION = 0;
+    std::vector<row_data> test_rows = generate_row_data();
+    test_rows[READ_HOT_PARTITION].get_qps = 5000.0;
+    test_rows[WRITE_HOT_PARTITION].put_qps = 5000.0;
+    for (int i = 0; i < FLAGS_occurrence_threshold; i++) {
+        aggregate_analyse_data(test_rows, calculator);
+    }
+    std::vector<std::array<int, 2>> expect_result = {
+        {0, 100}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {0, 0}, {100, 0}};
```

Comment:
Does the 100 has relative with the default value of ` FLAGS_occurrence_threshold`? If so, you should change it to ` FLAGS_occurrence_threshold`